src/test/java/ay2122s1_cs2103t_w16_2/btbb/model/order/OrderTest.java

```diff
@@ -1,53 +1,65 @@
 package ay2122s1_cs2103t_w16_2.btbb.model.order;
 
+import static ay2122s1_cs2103t_w16_2.btbb.logic.commands.CommandTestUtil.VALID_NAME_AMY;
 import static ay2122s1_cs2103t_w16_2.btbb.logic.commands.CommandTestUtil.VALID_PHONE_BOB;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_ALICE;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_BENSON;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_CARL;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_DANIEL;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_ELLE;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_ALICE;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_BENSON;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_CARL;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_DANIEL;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_ELLE;
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 import org.junit.jupiter.api.Test;
 
+import ay2122s1_cs2103t_w16_2.btbb.model.client.Address;
+import ay2122s1_cs2103t_w16_2.btbb.model.client.Name;
 import ay2122s1_cs2103t_w16_2.btbb.model.client.Phone;
 import ay2122s1_cs2103t_w16_2.btbb.testutil.OrderBuilder;
 
 class OrderTest {
     @Test
     public void isSameOrder() {
         // same object -> returns true
-        assertTrue(ORDER_BY_ALICE.equals(ORDER_BY_ALICE));
+        assertTrue(ORDER_FOR_ALICE.equals(ORDER_FOR_ALICE));
 
         // null -> returns false
-        assertFalse(ORDER_BY_BENSON.equals(null));
+        assertFalse(ORDER_FOR_BENSON.equals(null));
 
         // different phone -> returns false
```

Comment:
Can we make all the comments align? If it's for client use client X
```suggestion
        // different client phone -> returns false
```